lib/Services/MessageManager.vala

```diff
@@ -66,6 +76,10 @@ namespace Hemera.Services {
 
         public signal void receive_thinking ();
 
+        /**
+         * Parse JSON messages from Mycroft
+         * @return {@code void}
```

Comment:
I don't think we should use the valadoc `@return` field to specify the type, but rather for what is returned. I think it would look weird if there was something like the following in the documentation:
```c#
void readJSON (string json_message)
```
**returns:** `void`

Same thing below